src/Options/default_options.jl

```diff
@@ -67,6 +67,8 @@ function validate_solver_options_and_add_default_values!(options::Options)::Opti
     check_aliases_and_add_default_value!(dict, dict_copy, [:plot_vars, :output_variables], [0, 1])
     check_aliases_and_add_default_value!(dict, dict_copy, [:ε_proj], Inf)
     check_aliases_and_add_default_value!(dict, dict_copy, [:set_type_proj], Hyperrectangle)
+    check_aliases_and_add_default_value!(dict, dict_copy, [:global_vars], Vector{Int}())
```

Comment:
We have `vars` for CPost to specify variables we want to compute. Here I wanted to specify variables we want to compute for Hybrid Setting. Maybe `out_vars`?

Reply:
But why do we need them? Should the variables not only depend on the location (and its outgoing transition)?

Reply:
That's for the case if want to have some dimension in the output which is not constrained. For instance, Filtered Oscillator model has constrains only in x and y, but usually is is interested to get the plot on `x, x_n`.

Reply:
Then you can use `:plot_vars` aka `:output_variables`.

Reply:
`:plot_vars` has a restriction to have a length 2, while number of dimensions in the resulted set can be bigger (e.g. `x1,x2,x3` and we want to plot `x1,x2` and `x1,x3` without rerunning reachability). So, as I said before I just want to have `vars` option, but for hybrid setting.

Reply:
> `:plot_vars` has a restriction to have a length 2

That is unfortunate. Then `out_vars` is okay.
But I think it should be an option of the post operator(s) instead of a global option (because in other contexts it is ignored).

Reply:
> an option of the post operator(s) instead of a global option 

Hmm, I don't really like that we will have a mix of working with Discrete Post operator options in Continuous Post code. For now, you recommend to add `out_vars` as an option to `DecomposedDiscretePost`, which means I have to get this option in `post` of BFFPS19, filter with constrained variables and then add to `DecomposedDiscretePost` another option `temp_vars`. Even they are connected, it looks a bit messy:(

Reply:
Unfortunately, that is how the package is currently designed. I do not like it either, because the idea of decoupling is that things should not depend on each other, but in this case they do.
You can propose a new architecture, e.g., a `HybridAlgorithm` which consists of a `DiscretePost` and a `ContinuousPost`, which in this case would hold this option would be a clean solution. But this takes time.

> Even they are connected, it looks a bit messy:(

I find it messier to include algorithm-specific options as global options.

Reply:
okey, your point is valid:)